test_util/cluster_api.py

```diff
@@ -274,6 +274,7 @@ def get_test_app(self, custom_port=False):
             'cpus': 0.1,
             'mem': 32,
             'instances': 1,
+            'user': 'nobody',
```

Comment:
Would involve setting a `TEST_ADD_ENV_` in enterprise `util/run_ccm_test.sh`?
How would substitute in the correct expected user value based on security-mode?
Should I just read `default_task_user` out of bootstrap-config.json if it exists, and assume `root` otherwise?